sdk/appconfiguration/app-configuration/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - Specifying filters for listConfigurationSettings() or listRevisions() is 
   now done with the `keyFilter` or `labelFilter` strings rather than `keys`
   and `labels` as they were previously.
+- Fixes issue [#6408](https://github.com/Azure/azure-sdk-for-js/issues/6408)
```

Comment:
Can you give a little context on the issue? Like "Fixes issue where errors with a 429 statusCode weren't being retried"

Reply:
Done.